Software/Python/Examples/easy_Motors.py

```diff
@@ -37,8 +37,6 @@
 
 print("Drive the motors 50 cm and then stop.")
 gpg.drive_cm(50, True)
-
-print("Wait 1 second.")
```

Comment:
Well, every other command has the `sleep` function being called, so that's why I kept it. But at a 2nd thought, the `sleep` function was added to let the robot do its thing, so we remove the `sleep` too.

Reply:
Actually, I wouldn't want to remove it - I'd have to redo the GIF and it was time-consuming to synchronize both GIFs that go into it anyway.

Reply:
No problem. Put the print back in though so the user knows the robot is just waiting.